### PR TITLE
added the gif and updated the README.md file

### PR DESCRIPTION
**Pull request by Deepa-chaurasiya01**

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
Could you please make them a bit slow maybe. That looks pretty flashy. No need to record entire level. Just give a preview of that level. That would work fine too.

**Comment by Deepa-chaurasiya01** [comment]
okay sir

File: README.md [hunk 1]
```diff
@@ -114,8 +117,11 @@ The story of the project is pretty interesting. You are a famous magician in the
       <div class = "column">
         <img src = "assets/Screenshots/Flying Wizard/ss23.png" height = "200px">
       </div>
-  </div>
+  </div> -->
+
 
+<img src = "assets/Screenshots/gif/Flying-w/Flying-wizard.gif
```
Comment (kaiwalyakoparkar) [inline README.md:123]: This links seems to be broken, Could you please fix it.
[Ref:]

File: README.md [hunk 2]
```diff
@@ -95,17 +95,20 @@ The story of the project is pretty interesting. You are a famous magician in the
...
+
+
+  <img src = "assets/Screenshots/gif/Witch.gif">
+
+  <img src = "assets/Screenshots/gif/potions.gif">
 
 ## 🚩Flying Wizard:
 
- <div class = "row">
+ <!-- <div class = "row">
```
Comment (kaiwalyakoparkar) [inline README.md:108]: Please remove this commented code :)

File: README.md [hunk 3]
```diff
@@ -69,12 +69,12 @@ The story of the project is pretty interesting. You are a famous magician in the
...
  -->
 
-## 📌 [Screenshots](/assets/Screenshots)
+## 📌 [Project-preview](/assets/Screenshots)
 
 
 ## 🚩 Escape-room:
 
-  <div class = "row">
+  <!-- <div class = "row">
```
Comment (kaiwalyakoparkar) [inline README.md:77]: Please remove this commented code.

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by Deepa-chaurasiya01** [comment]
Sorry ! Sir, I was checking the gifs. I'm sorry!!

**Pull request closed without merging**